### DISCUSSION TITLE
localStorage is not defined

### DISCUSSION THREAD

**Discussion by izhaorui** [vuepress/core#1325] [Q&A] [answered]
在进行 vuepress build src 打包时，一些API使用不了，比如 location、window、localStorage等，在dev模式下都是正常的

**Comment by Mister-Hope** [comment, marked as answer]
https://vuejs.press/advanced/cookbook/usage-of-client-config.html#use-non-ssr-friendly-features-1

**Reply by izhaorui** [reply]
加了还不行
<img width="364" alt="image" src="https://user-images.githubusercontent.com/16451400/236822226-f3f8fe44-bef8-4372-b575-e690211d5f41.png">

<img width="689" alt="image" src="https://user-images.githubusercontent.com/16451400/236822327-41e1e5fe-d21b-440e-a65e-0817f4ed1af3.png">
